pages/categories/_single.vue

```diff
@@ -39,6 +39,17 @@ export default {
       allCats: []
     }
   },
+  head() {
+    return {
+      meta: [
+        {
+          hid: 'og:image',
+          property: 'og:image',
+          content: this.$store.state.image
```

Comment:
@danielkellyio I'm not really sure what to do for this one... One of your demo categories is a full URL, and the other is a "/logo.svg" which needs the full URL to render on an OpenGraph card...

Honestly, passing the metadata via each page feels a little off. Shouldn't it all be handled by `Default.vue`? I haven't really done much work in Vue before :/

Reply:
what's handled in the default.vue really is just for the posts (which does kind of feel wrong for sure). Part of the problem, is that I've named the image property one thing categories and another for posts ( image vs featureImage). Probably the best way to go about solving this for content that comes from the CMS is to name all the main image properties the same (and then it could all be done through default.vue)

As for the contact and about pages they would have to be done at the page level since their contact isn't defined in the store but hardcoded in the page.

Reply:
Ah, I see. I'll look into that :)